packages/teams-js/test/public/media.spec.ts

```diff
@@ -442,58 +444,72 @@ describe('media', () => {
           } as DOMMessageEvent);
         });
       });
-
-      // TODO: fix bug and include
-      it.skip('selectMedia calls with successful result for mediaType = 1 with imageOutputFormats', () => {
-        return mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android).then(() => {
+      it('selectMedia calls with successful result for mediaType = 1 with imageOutputFormats', done => {
+        mobilePlatformMock.initializeWithContext(FrameContexts.content, HostClientType.android).then(() => {
           mobilePlatformMock.setClientSupportedSDKVersion(imageOutputFormatsAPISupportVersion);
-          let mediaAttachments: media.Media[], mediaError: SdkError;
           const mediaInputs: media.MediaInputs = {
-            mediaType: media.MediaType.Image,
+            mediaType: media.MediaType.VideoAndImage,
```

Comment:
Was it intentional to change Image into VideoAndImage for the merge? If so, does it not work if with Image?

Reply:
i think this was a mistake, good catch!